src/applications/hca/routes.jsx

```diff
@@ -1,13 +1,33 @@
-import { createRoutesWithSaveInProgress } from '../../platform/forms/save-in-progress/helpers';
+import { createRoutesWithSaveInProgress } from 'platform/forms/save-in-progress/helpers';
+import environment from 'platform/utilities/environment';
 
 import formConfig from './config/form';
 import HealthCareApp from './HealthCareApp.jsx';
+import IDPage from './containers/IDPage';
 
-const route = {
+const formRoutes = createRoutesWithSaveInProgress(formConfig);
+
+// If the route's path ends in `introduction` then the form system will treat it
+// as an Introduction page (ie it won't add the Title component or navigation
+// elements) which is what we want in this case.
+const idFormRoute = {
+  path: 'id-introduction',
+  component: IDPage,
+  key: 'id-introduction',
+};
+
+// do not allow the idFormRoute in production
+if (environment.isProduction()) {
```

Comment:
It seems like you'd want to remove the `<IDForm>` component from the intro page if you were trying to gate this from going to production, rather than trying to mess with re-routing.

Reply:
Or just don't add this route as a child route in production? I agree that messing with routing isn't the best way to handle this.

Reply:
Yeah, I think both skipping the child route and not including the component in the intro page would work for gating it.

Reply:
Is removing the `IDForm` from the `IDPage` just an extra safety measure? Because if the entire `IDPage` route isn't available on prod, it seems unnecessary. I'm just trying to understand your thinking here.

Reply:
I might be misunderstanding how this works, but I'm assuming that something related to the IDForm component will display on the intro page if you don't remove it there.

Reply:
There is no `<IDForm>` on the standard Introduction page, it's only on this additional `id-introduction` route. The new route is additive, not a replacement for the existing `introduction` route.

Reply:
Oh, gotcha, I misread it. Yeah, just removing the route is good, I think.